middleware/createAccount.js

```diff
@@ -0,0 +1,25 @@
+const { creatorKeyJson } = require('./near');
+
+const NEW_ACCOUNT_AMOUNT = process.env.NEW_ACCOUNT_AMOUNT;
+
+const createAccount = async (ctx) => {
+    if (!creatorKeyJson) {
+        console.warn('ACCOUNT_CREATOR_KEY is not set up, cannot create accounts.');
+        ctx.throw(500, 'Service misconfigured; account creation is not available.');
+    }
+
+    const { newAccountId, newAccountPublicKey } = ctx.request.body;
+
+    if (newAccountId.includes('dzarezenko')) {
```

Comment:
time to remove this check, he already stopped trolling us I believe :)

Reply:
:+1: killing it now